modules/ROOT/pages/connected-apps-overview.adoc

```diff
@@ -16,6 +16,25 @@ OAuth (Open Authorization) is an open protocol that provides secure API authoriz
 OAuth can authorize access to resources without revealing user credentials to apps.
 OpenID Connect identifies the end user and obtains information to pass to OAuth 2.0 connected apps.
 
+
+// Sara add some indentation or mark up here please
+
+In order to understand the purpose of Connected Apps, there are several technical concepts and terms to be familiar with, beginning with OAuth 2.0 (commonly referred to as just OAuth) and OpenID Connect (commonly referred to as OICD). Before breaking down these terms, it’s important to understand that these flows are completed in order to give access to Anypoint APIs to only those services and users with the proper permissions. 
```

Comment:
This seems less about Connected Apps directly and more about the authentication aspect of it. I'll be running under this assumption herein.

Reply:
```suggestion
The Connected Apps feature uses OAuth 2.0 and OpenID Connect, or _OIDC_, as part of its authentication process. OAuth 2.0 and OpenID Connect flows must be completed in order to grant access to Anypoint APIs only to services and users that have the proper permissions. 
```

Reply:
We do not want to refer to OAuth 2.0 as OAuth for short. OAuth 2.0 is a completely revamped version of OAuth, so I'll be scouting this out in docs and fixing it. I think this is wrong in a few legacy docs too.